src/github/commit/index.ts

```diff
@@ -32,10 +32,7 @@ interface IGitHubCommitArgs {
  */
 export const execute = async (args: IGitHubCommitArgs = {}): Promise<string | undefined> => {
   let retVal: string | undefined =
-    process.env.GITHUB_SHA ||
-    process.env.CIRCLE_SHA1 ||
-    process.env.TRAVIS_PULL_REQUEST_SHA ||
-    process.env.TRAVIS_COMMIT;
+    process.env.CIRCLE_SHA1 || process.env.TRAVIS_PULL_REQUEST_SHA || process.env.TRAVIS_COMMIT;
```

Comment:
```suggestion
    process.env.COMMIT_SHA ||
    process.env.GITHUB_SHA ||
    process.env.CIRCLE_SHA1 || 
    process.env.TRAVIS_PULL_REQUEST_SHA || 
    process.env.TRAVIS_COMMIT;
```

Oh 😮 so based on https://github.com/zendeskgarden/scripts/pull/228#issuecomment-2224045637 can this suggestion work? If I'm reading you right, this is the same as what we had with Travis.